### PR TITLE
add logo for custstats custom integration

### PR DESCRIPTION
**Pull request by hekmon**
## Proposed change

Add icons for [custstats](https://github.com/hekmon/ha-custstats) integration. `custstats` or `Home Assistant Custom Integration Statistics` is a small integration allowing to recover custom integration installation statistics from [Home Assistant official custom integration statistics](https://analytics.home-assistant.io/custom_integrations.json) and integrate them into HA.

As the service provider for this integration is Home Assistant itself, I chose the Home Assistant logo (as icon). I am not aware of official HA logo that could fit the logo part so I did not include one.

Feel free to discuss/request change for this integration usage of HA logo.

## Type of change

- [ ] Add a new logo or icon for a new core integration
- [ ] Add a missing icon or logo for an existing core integration
- [x] Add a new logo or icon for a custom integration (custom component)
  - [ ] I've opened up a PR for my custom integration on the [Home Assistant
    Python wheels repository](https://github.com/home-assistant/wheels-custom-integrations)
- [ ] Replace an existing icon or logo with a higher quality version
- [ ] Removing an icon or logo

## Additional information

- This PR fixes or closes issue: fixes #
- Link to code base pull request: 
- Link to documentation pull request: 
- Link to integration documentation on our website: 

## Checklist

- [x] The added/replaced image(s) are **PNG**
- [x] Icon image size is 256x256px (`icon.png`)
- [x] hDPI icon image size is 512x512px for  (`icon@2x.png`)
- [ ] Logo image size has min 128px, but max 256px, on the shortest side (`logo.png`)
- [ ] hDPI logo image size has min 256px, but max 512px, on the shortest side (`logo@2x.png`)

### CODE REVIEW

**Review by frenck — CHANGES_REQUESTED** [review]
We do not allow the use of our logo for such use cases, as it might confuse the end-user it is officially supported.

**Comment by hekmon** [comment]
I thought as much!

What would you recommend in that case? I am free to come up with my own?

**Comment by frenck** [comment]
I guess so

**Comment by hekmon** [comment]
had some help from my mdi friend, I guess that should do

**Review by frenck — CHANGES_REQUESTED** [review]
Please adjust the color to something else, we use this color to mark our helpers. We want to avoid unsupported custom integration to be identified as supported or provided by the Home Assistant project.

../Frenck

**Comment by hekmon** [comment]
Is this blue acceptable ?

**Review by frenck — APPROVED** [review]
I think you are pretty much trying to look for the edge of allowance, can be avoid any discussion and just make it a completely different color?

**Review by frenck — CHANGES_REQUESTED** [review]
Wrong button let me try again.

**Review by frenck — APPROVED** [review]
Thanks, @hekmon 👍 

../Frenck

**Comment by hekmon** [comment]
I honestly tought that the deep signal blue was different enough from the light home assistant blue. But hey, you are the boss :)

This light green should not conflict with anything from ha by far !

**Comment by hekmon** [comment]
One last thing @frenck , do you know how often the [stats page](https://analytics.home-assistant.io/custom_integrations.json) is updated ?

I set the [polling](https://github.com/hekmon/ha-custstats/blob/v1.0.0/custom_components/custstats/sensor.py#L17) to 1 hour but it would be best to set it to the exact upstream refresh interval (and avoid polling it every hour is the upstream is updated only every 6 hours for example).

**Comment by frenck** [comment]
<https://github.com/home-assistant/analytics.home-assistant.io#schedule>

**Comment by hekmon** [comment]
Every hour is fine then, thank you.
